packages/twenty-server/src/modules/workflow/workflow-trigger/utils/assert-version-can-be-activated.util.ts

```diff
@@ -37,11 +32,7 @@ export function assertVersionCanBeActivated(
   }
 }
 
-function assertVersionIsValid(
-  workflowVersion: Omit<WorkflowVersionWorkspaceEntity, 'trigger'> & {
-    trigger: WorkflowTrigger;
-  },
-) {
+function assertVersionIsValid(workflowVersion: WorkflowVersionWorkspaceEntity) {
```

Comment:
same :)

Reply:
I think it something we wanted to do on the BE too
@Weiko @magrinj could you confirm?

I feel it's more clear personnally

Reply:
We can discuss this topic asynchronously and fix the problem globally once we decide.